Justfile

```diff
@@ -40,7 +40,7 @@ build target=default-target:
     {{ cargo-cmd }} build --profile={{ if target == "debug" { "dev" } else { target } }} {{ target-triple-flag }}
 
 # build testing guest binaries
-guests: build-and-move-rust-guests build-and-move-c-guests
+guests target=default-target: (build-and-move-rust-guests target) (build-and-move-c-guests target)
```

Comment:
benchmark.yml requires both debug and release to be built by `build-guest-binaries`. Maybe it also needs updating?

Reply:
ah, i think i messed this up.  Im just going to drop this commit as i think it worked better prior

Reply:
dropped, so there is no change here.